AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/AWSModelReconciliationQueue.swift

```diff
@@ -64,7 +64,7 @@ final class AWSModelReconciliationQueue: ModelReconciliationQueue {
     private let modelName: String
 
     private var incomingEventsSink: AnyCancellable?
-    private var reconcileAndLocalSaveOperationSink: AnyCancellable?
+    private var reconcileAndLocalSaveOperationSinks = Set<AnyCancellable?>()
```

Comment:
nittiest of nits: I prefer to reserve variable assignments outside of initializers for types that don't have an initializer. Otherwise, a reader has to look in two places to know what the initial state of an object is.

Reply:
Done.